package_name: pr2_dmp_data
data_directory_name: dmp_data